src/main/java/org/cascadebot/cascadebot/utils/votes/VoteButtonGroupBuilder.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.utils.votes;
+
+import net.dv8tion.jda.core.entities.Emote;
+import net.dv8tion.jda.core.entities.Message;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.utils.buttons.Button;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+public class VoteButtonGroupBuilder {
+
+    private VoteMessageType type;
+
+    private Timer timer = new Timer();
+
+    private List<Button> extraButtonList = new ArrayList<>();
+    private List<Object> voteButtons = new ArrayList<>();
+    private int amount = 1; //Setting this so things don't break
+
+    private long voteTime = TimeUnit.SECONDS.toMillis(30);
+
+    private Consumer<List<VoteResult>> finishConsumer;
+
+    private IVotePeriodicRunnable periodicConsumer;
```

Comment:
Why is this named consumer if it's not a consumer?

Reply:
Also when is this run?

Reply:
too lazy to rename. it runs every 5 seconds

Reply:
Well rename it, idc if you're lazy :P